geode-core/src/test/java/org/apache/geode/distributed/LocatorDUnitTest.java

```diff
@@ -311,9 +313,7 @@ private Boolean startLocatorWithPortAndProperties(final int port, final Properti
       throws IOException {
     try {
       Locator.startLocatorAndDS(port, new File(""), properties);
-    } catch (SystemConnectException e) {
-      return Boolean.FALSE;
-    } catch (GemFireConfigException e) {
+    } catch (SystemConnectException | GemFireConfigException e) {
```

Comment:
Nothing seems to be using this return value. So it seems like we a swallowing an exception here?

Reply:
Sure enough. I'm making this return void and not catching the exception.